src/uct/ib/base/ib_iface.c

```diff
@@ -614,9 +616,12 @@ uct_ib_iface_roce_is_reachable(const uct_ib_device_gid_info_t *local_gid_info,
 
     if (local_ib_addr_af != remote_ib_addr_af) {
         ucs_assert(local_ib_addr_af != 0);
-        ucs_debug("different addr_family detected. local %s remote %s",
-                  ucs_sockaddr_address_family_str(local_ib_addr_af),
-                  ucs_sockaddr_address_family_str(remote_ib_addr_af));
+        if ((info_string != NULL) && (info_string_length > 0)) {
```

Comment:
To avoid code duplication all input parameters checks could be done in common function `uct_iface_is_reachable_v2`.
Also, if `field_mask` has initialized bits, we assume that input parameters are correct.
All additional check for correctness of value, for example, should be done by `UCT_CHECK_PARAM` macro or similar.

Reply:
Done. Use UCT_CHECK_PARAM in uct_iface_is_reachable_v2 and remove all `if ((info_string != NULL) && (info_string_length > 0))`